### ISSUE TITLE
Docker-compose lacks env vars needed to change default credentials

### ISSUE THREAD

**Issue by Scronkfinkle** [wazuh/wazuh-docker#823]
I've been scratching my head for the past couple days trying to figure out why changing the `kibanaserver` password in the indexer causes the dashboard to crash. Here's how to reproduce the issue and how I have fixed it.    

First, I followed the instructions here https://documentation.wazuh.com/current/deployment-options/docker/wazuh-container.html ,   
and repeated the "Change the password of an existing user" section for all the default users in the `internal_users.yml`. The, I ran `securityadmin.sh` and restarted my containers with `docker-compose down && docker-compose up -d`.  At this point, the dashboard hangs at the message `Wazuh dashboard server is not ready yet`. Looking at the logs  

`docker logs single-node_wazuh.dashboard_1`:
```
"type":"log","@timestamp":"2023-04-18T13:42:30Z","tags":["error","opensearch","data"],"pid":39,"message":"[ResponseError]: Response Error"
```

`docker logs single-node_wazuh.indexer_1`
```
[2023-04-18T13:42:18,464][WARN ][o.o.s.a.BackendRegistry  ] [wazuh.indexer] Authentication finally failed for kibanaserver from 192.168.176.4:41534
```

I noticed that over in kubernetes-land they have an additional env var for configuring this user, `DASHBOARD_USERNAME` and `DASHBOARD_PASSWORD`. It turns out after setting 
```
- DASHBOARD_USERNAME=kibanaserver
- DASHBOARD_PASSWORD=<new_password>
```
in the `environment` section for the dashboard, Wazuh begins working again as expected.

**Comment by nu-wa** [comment]
I encountered the same problem a few months back, I only thought of adding it to the documentation, not here.

Here's the PR for the documentation in case this gets merged :) wazuh/wazuh-documentation#5733

**Comment by vcerenu** [comment]
Hi @Scronkfinkle 
I will be responding in the PR to the changes that we may need for the merge. Thanks!!

Hi @nu-wa 
I'm reviewing your PR with the authorized area for the generation of documentation. Thanks!!
